src/utils/normalizeConfig.js

```diff
@@ -111,13 +111,15 @@ const normalizeCliOptions = rawConfig =>
 
     const value = rawConfig[key] !== undefined ? rawConfig[key] : defaultValue;
 
+    // eslint-disable-next-line prefer-object-spread
```

Comment:
it should either be disabled in the whole project (if it supports a node without spread), or it should be fixed here.